AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/HierarchicalVersionUtils.dfy

```diff
@@ -43,10 +43,10 @@ module HierarchicalVersionUtils {
     return Success(Crypto);
   }
 
-  method createMdDigest (
+  method createBKCDigest (
     branchKeyContext: map<string, string>,
     cryptoClient: AtomicPrimitives.AtomicPrimitivesClient
-  ) returns (output: Result<seq<uint8>, Types.Error>)
+  ) returns (output: Result<seq<uint8>, Types.KeyStoreException>)
```

Comment:
OH, this is not what I meant; this needs to be Types.Error,
but you can do something like:
```
type BKCDigestError = e: Types.Error | (e.KeyStoreException? ) witness *
```
And then you the method can return:
```
) returns (output: Result<seq<uint8>, BKCDigestError>)
```
But there does not seem to be much value in that here,
since we cannot really create a good error for this.

See this for an example: https://github.com/aws/aws-cryptographic-material-providers-library/blob/hv-2/hv-2/AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/src/CanonicalEncryptionContext.dfy

Reply:
Done